ctapipe/configuration/test/TestConfiguration.py

```diff
@@ -0,0 +1,200 @@
+#/bin/env python3
```

Comment:
Use pytest (not unittest) and follow these testing guidelines:
http://astropy.readthedocs.org/en/latest/development/testguide.html

It means re have to re-write this quite a bit:
- File name should be `test_core.py`
- `#!` line and docstring at the top and `__main__` part at the bottom can be removed
- Test asserts have to be rewritten to just use asserts like is common in pytest (check astropy tests for examples)